### DISCUSSION TITLE
yarp.it 403 Forbidden

### DISCUSSION THREAD

**Discussion by jgvictores** [robotology/community#58] [Q&A] [answered]
"Forbidden
You don't have permission to access / on this server."
It's been like that for a couple of days... Bad, because yarp.it is cool and it is also linked from https://en.wikipedia.org/wiki/YARP

**Comment by mbrunettini** [comment, marked as answer]
we're re-organizing web services: the DNS has been updated to point to the new one.
It needs a while before the DNS caches update.
